README.md

```diff
@@ -19,99 +19,53 @@ You can watch videos of the system being developed on YouTube:
 * [Andreas Kling's channel](https://youtube.com/andreaskling)
 * [Linus Groh's channel](https://youtube.com/linusgroh)
 
+**FAQ**: [Frequently Asked Questions](https://github.com/SerenityOS/serenity/blob/master/Documentation/FAQ.md)
+
 ## Screenshot
 
 ![Screenshot as of b36968c.png](https://raw.githubusercontent.com/SerenityOS/serenity/master/Meta/screenshot-b36968c.png)
 
-## Kernel features
-
-* x86 (32-bit) and x86_64 (64-bit) kernel with pre-emptive multi-threading
-* Hardware protections (SMEP, SMAP, UMIP, NX, WP, TSD, ...)
-* IPv4 stack with ARP, TCP, UDP and ICMP protocols
-* ext2 filesystem
-* POSIX signals
-* Purgeable memory
-* /proc filesystem
-* Pseudoterminals (with /dev/pts filesystem)
-* Filesystem notifications
-* CPU and memory profiling
-* VMWare/QEMU mouse integration
-
-## System services
-
-* Launch/session daemon (SystemServer)
-* Compositing window server (WindowServer)
-* Text console manager (TTYServer)
-* DNS client (LookupServer)
-* Network protocols server (RequestServer and WebSocket)
-* Software-mixing sound daemon (AudioServer)
-* Desktop notifications (NotificationServer)
-* HTTP server (WebServer)
-* Telnet server (TelnetServer)
-* DHCP client (DHCPClient)
-
-## Libraries
-
-* C++ templates and containers (AK)
-* Event loop and utilities (LibCore)
-* 2D graphics library (LibGfx)
-* OpenGL 1.x compatible library (LibGL)
-* GUI toolkit (LibGUI)
-* Cross-process communication library (LibIPC)
-* HTML/CSS engine (LibWeb)
-* JavaScript engine (LibJS)
-* Markdown (LibMarkdown)
-* Audio (LibAudio)
-* Digital Signal Processing/Synthesizer Chains (LibDSP)
-* PCI database (LibPCIDB)
-* Terminal emulation (LibVT)
-* Out-of-process network protocol I/O (LibProtocol)
-* Mathematical functions (LibM)
-* ELF file handling (LibELF)
-* POSIX threading (LibPthread)
-* Higher-level threading (LibThreading)
-* Transport Layer Security (LibTLS)
-* HTTP and HTTPS (LibHTTP)
-* IMAP (LibIMAP)
-
-## Userland features
-
-* Unix-like libc and userland
-* Shell with pipes and I/O redirection
-* On-line help system (both terminal and GUI variants)
-* Web browser (Browser)
-* C++ IDE (HackStudio)
-* Desktop synthesizer (Piano)
-* E-mail client (Mail)
-* Various desktop apps & games
-* Color themes
+## Features
+* Modern x86 32-bit and 64-bit kernel with pre-emptive multi-threading
+* [Browser](Userland/Applications/Browser/) with JavaScript, WASM and more (check the spec compliance [here](https://libjs.dev/test262/), [here](https://css.tobyase.de/) and [here](https://libjs.dev/wasm/))
```

Comment:
More explicit labels for the links rather than "here":
```suggestion
* [Browser](Userland/Applications/Browser/) with JavaScript, WASM and more (check the spec compliance for [JavaScript](https://libjs.dev/test262/), [CSS](https://css.tobyase.de/) and [Wasm](https://libjs.dev/wasm/))
```

Reply:
I think it's kind of unexpected to have compliance links behind the simple words when all other terms link to the code.

Reply:
Not sure we need spec compliance link here at all.

Reply:
Because a highly compliant browser is orders of magnitude better and more impressive than a Browser that can just handle some \<p\>'s, I'd say!